sim/team-validator.ts

```diff
@@ -1374,6 +1378,28 @@ export class TeamValidator {
 			if (banReason === '') return null;
 		}
 
+		if (dex.species.get(species.baseSpecies).tags?.length) {
+			if (dex.species.get(species.baseSpecies).tags.includes("Mythical")) {
```

Comment:
nit: you could extract tags into a const to save getting it multiple times.

Reply:
we could add it in line 1290